README.md

```diff
@@ -52,63 +61,35 @@ console.log(result.response.text());
 
 ## Try out a sample app
 
-This repository contains sample Node and web apps demonstrating how the SDK can access and utilize the Gemini model for various use cases.
+This repository contains sample Node and web apps demonstrating how the SDK can
+access and utilize the Gemini model for various use cases.
 
 **To try out the sample Node app, follow these steps:**
 
-1.  Check out this repository.\
-`git clone https://github.com/google/generative-ai-js`
+1.  Check out this repository. \
+    `git clone https://github.com/google/generative-ai-js`
 
-1.  [Obtain an API key](https://makersuite.google.com/app/apikey) to use with the Google AI SDKs.
+1.  [Obtain an API key](https://makersuite.google.com/app/apikey) to use with
+    the Google AI SDKs.
 
 1.  cd into the `samples/node` folder and run `npm install`.
 
 1.  Assign your API key to an environment variable: `export API_KEY=MY_API_KEY`.
 
 1.  Run the sample file you're interested in. Example: `node simple-text.js`.
 
-**To try out the sample web app, follow these steps:**
-
-1.  Check out this repository.\
-`git clone https://github.com/google/generative-ai-js`
-
-1.  [Obtain an API key](https://makersuite.google.com/app/apikey) to use with the Google AI SDKs.
-
-1.  cd into the `samples/web` folder and run `npm install`.
-
-1.  Assign your API key to an environment variable: `export API_KEY=MY_API_KEY`.
-
-1.  Serve your web app by running: `npm run http-server`. Open the displayed URL in a browser.
-
-## Installation and usage
-
-- For Node.js (or web projects using NPM), run `npm install @google/generative-ai`.
-- For web, add `import  { GoogleGenerativeAI } from "https://esm.run/@google/generative-ai"`.
-
-For detailed instructions, you can find quickstarts for the Google AI JavaScript SDK in the Google documentation:
-
-
-- [Quickstart for Node.js](https://ai.google.dev/tutorials/node_quickstart)
-- [Quickstart for web apps](https://ai.google.dev/tutorials/web_quickstart)
-
-These quickstarts describe how to add your API key and the SDK to your app, initialize the model, and then call the API to access the model. It also describes some additional use cases and features, like streaming, counting tokens, and controlling responses. For Node.js, embedding is also available.
-
 ## Documentation
 
-Find complete documentation for the Google AI SDKs and the Gemini model in the Google documentation:\
-https://ai.google.dev/docs
-
-Find reference docs for this SDK here in the repo:
-- [GoogleGenerativeAI](/docs/reference/main/generative-ai.md)
```

Comment:
Sorry I missed this earlier, do we need to remove the reference docs links? This is the only place there are reference docs for the JS SDK, there's no reference docs available (for the SDK) on ai.google.dev. And they're a little hard to find just by looking at the repo directory structure.

Reply:
That's a good point. I'll find a way to put it back.